src/main/java/supertracker/command/AddCommand.java

```diff
@@ -8,16 +8,38 @@
 
 import java.io.IOException;
 
+/**
+ * Represents a command for increasing the quantity of an item.
+ */
 public class AddCommand implements Command {
     protected String name;
     protected int quantity;
     protected Item newItem;
 
+    /**
+     * Constructs a new AddCommand object with the specified name and quantity.
+     *
+     * @param name     Name of the item to be added
+     * @param quantity Quantity of the item to be added
+     */
     public AddCommand(String name, int quantity) {
         this.name = name;
         this.quantity = quantity;
     }
 
+    /**
+     * Executes the AddCommand without displaying any user interface.
+     * <p>
+     * This method adds the specified quantity of an item to the inventory. It ensures
+     * that the inventory contains the specified item and that the quantity is non-negative.
+     * If the item already exists in the inventory, its quantity is updated accordingly.
+     * After updating the inventory, the changes are saved to persistent storage.
+     * If an IOException occurs while saving data, an error message is printed.
+     * </p>
```

Comment:
I believe the closing of the p in html is unnecessary. Can consider removing it. This applies for the other instances this happens for your javadocs.

My reference: https://nus-cs2113-ay2324s2.github.io/website/schedule/week7/topics.html#javadoc